doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -185,6 +185,7 @@ The ``bt_fast_pair`` partition address is provided automatically by the build sy
 For example, when building an application with the |nRFVSC|, you need to add the following parameters in the **Extra CMake arguments** field on the **Add Build Configuration view**: ``-DFP_MODEL_ID=0xFFFFFF -DFP_ANTI_SPOOFING_KEY=AbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbA=``.
 Make sure to replace ``0xFFFFFF`` and ``AbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbA=`` with values obtained for your device.
 See :ref:`cmake_options` for more information about defining CMake options.
+See the following sections for information on how to add the Google Fast Pair subsystem to your project.
```

Comment:
Could you please use this sentence directly after the sentence from L177?

_The build system automatically calls the Fast Pair provision script and includes the resulting hexadecimal file in the firmware (the :file:`merged.hex` file) when building with Fast Pair in the |NCS|._